BuildResidentialHPXML/measure.rb

```diff
@@ -3448,9 +3462,9 @@ def self.create(runner, model, args, epw_path, hpxml_path)
       end
 
       eri_version = Constants.ERIVersions[-1]
-      # FIXME: Address this when multiple buildings
-      HPXMLDefaults.apply(runner, hpxml, hpxml.buildings[0], eri_version, weather, epw_file: epw_file)
+      HPXMLDefaults.apply(runner, hpxml, hpxml_bldg, eri_version, weather, epw_file: epw_file)
       hpxml_doc = hpxml.to_doc()
+      hpxml.set_unique_hpxml_ids(hpxml_doc, true) if hpxml.buildings.size > 1
```

Comment:
`hpxml.set_unique_hpxml_ids` is also called above... if `apply_defaults` is true and we get to here, won't this change awall with the name "wall2" to "wall22"?

Reply:
Hmm I don't think so, because in the line above we are making a new `hpxml_doc` that gets passed into `hpxml.set_unique_hpxml_ids`...

Reply:
(And if `hpxml.set_unique_hpxml_ids` were called twice in a row on the same `hpxml_doc`, wouldn't it be something like "wall2" to "wall2_2_2"?)